gamemodes/terrortown/gamemode/server/sv_player_ext.lua

```diff
@@ -682,6 +682,10 @@ function plymeta:InitialSpawn()
 
 	self:SetPos(spawnPoint.pos)
 	self:SetAngles(spawnPoint.ang)
+
+	-- register synced vars
+	self:RegisterSettingOnServer("enable_dynamic_fov", "bool")
+	self:RegisterSettingOnServer("fov_desired", "float")
```

Comment:
I think it would be easier to not do that for each player and maybe handle this globally?

Reply:
Maybe, not so sure. But as the setting is ok the player it might be interesting to allow different kinds of settings for different players.

E.g. an admin could have more synced settings

Reply:
Yes, but since this is also a case of "could/might" and it is not necessary for now we should not make it more complicated, as it can easily be adjusted again in the future to do so. 

Just feels more hacky having to set this on initial spawn on every player and for other people trying to use this it is also weird because they need to initialize it on every player.